qdax/utils/metrics.py

```diff
@@ -61,10 +92,49 @@ def default_qd_metrics(
     """
 
     # get metrics
-    grid_empty = repertoire.fitnesses == -jnp.inf
-    qd_score = jnp.sum(repertoire.fitnesses, where=~grid_empty)
-    qd_score += qd_offset * jnp.sum(1.0 - grid_empty)
-    coverage = 100 * jnp.mean(1.0 - grid_empty)
+    repertoire_empty = repertoire.fitnesses == -jnp.inf
+    qd_score = jnp.sum(repertoire.fitnesses, where=~repertoire_empty)
+    qd_score += qd_offset * jnp.sum(1.0 - repertoire_empty)
+    coverage = 100 * jnp.mean(1.0 - repertoire_empty)
     max_fitness = jnp.max(repertoire.fitnesses)
 
     return {"qd_score": qd_score, "max_fitness": max_fitness, "coverage": coverage}
+
+
+def default_moqd_metrics(
+    repertoire: MOMERepertoire, reference_point: jnp.ndarray
+) -> Dict[str, jnp.ndarray]:
+    """
+    Compute the MOQD metric given a MOME repertoire and a reference point.
+    """
```

Comment:
Maybe this DocString could be more detailed? 
What is the `reference_point` used for?